js/script.js

```diff
@@ -0,0 +1,309 @@
+// } else if(month >=6 && month <=8){
+//   alert('summer');
+// } else if(month >=9 && month <=11){
+//   alert('autumn');
+// }
+
+// let str = 'abcde';
+// if (str[0] == 'a') {
+// alert('Да');
+// } else {
+// alert('Нет');
+// }
+
+// let str = '123';
+// let sum = Number(str[0]) + Number(str[1]) + Number(str[2]);
+// alert(sum);
+
+// let str = '123456';
+// let str1 = Number(str[0]) + Number(str[1]) + Number(str[2]);
+// let str2 = Number(str[3]) + Number(str[4]) + Number(str[5]);
+// if (str1 == str2) {
+// alert('Да');
+// } else {
+// alert('Нет');
+// }
+
+// let str = 'abcde';
+// alert(str[0]);
+// alert(str[1]);
+// alert(str[4]);
+
+// let a = prompt('day','');
+// alert(60 * 60 * 24 * +a +' seconds');
+
+// let num = 1;
+// num = num + 12;
+// num = num - 14;
+// num = num * 5;
+// num = num / 7;
+// num = num + 1;
+// num = num - 1;
+// alert(num);
+
+// let num = 1;
+// num += 12;
+// num -= 14;
+// num *= 5;
+// num /= 7;
+// num += 1;
+// num -= 1;
+// alert(num);
+
+// let str1 = 'Hello, ';
+// let str2 = 'World!';
+// alert(str1 + str2);
+
+// let name ='Evgeniy';
+// alert('Hello,' + name);
+
+// let num = prompt('Enter the num','');
+// alert(num*num);
+
+// let num = '12345';
+// alert(num[0] * num[1] * num[2] * num[3] * num[4]);
+
+//==========Funcion==========//
+
+
+// function min(a, b) {
+//   if (a > b) {
+//     return 1;
+//   } else if(a<b){
+//     return -1;
+//   } else(a=b)
+//   return 0;
+// }
+// console.log(min(3,3));
+
+
+// showWords = function() {
+//   let args = [...arguments];
+//   document.write(args.join(' '));
+// }
+// showWords("I've", "been", "learning", "JavaScript", "for", "a", "month");
+
+// //второй вариант, как лучше?
+// showWords = function() {
+//   let result = '';
+//   let n = arguments.length;
+//   for(let i=0; i<n; i++) {
+//     result += arguments[i] + " ";
+//   }
+//   document.write(result);
+// }
+// showWords("I've", "been", "learning", "JavaScript", "for", "a", "month");
+
+// complexDigit = function() {
+//   const args = [...arguments];
+//   document.write(args.join(''));
+// }
+// complexDigit(3,5,6,1,3,4,5);
+
+//==========Объекты==========//
+
+// const user = {
+//   name: 'John',
+//   surname: 'Smith',
+//   name: 'Pete',
+// };
+// delete user.name;
+// console.log(user);
+
+// const salaries = {
+//   John: 100,
+//   Ann: 160,
+//   Pete: 130,
+// };
+// let sum = 0;
+// for(let key in salaries){
+//   sum += (salaries[key]);
+// }
+// console.log(sum);
+
+
+// let menu ={
+//   width: 200,
+//   height: 300,
+//   title: "My menu",
+// };
+// console.log(menu);
+// myltiplyNumeric(menu);
+// function myltiplyNumeric(obj) {
+//   for(let key in obj) {
+//     if(typeof obj[key] == 'number'){
+//       obj[key] *= 2;
+//     }
+//   }
+// }
+// console.log(menu);
+
+// let user = {
+//   name: "John",
+//   age: 30,
+// };
+// console.log(user.age);
+
+// const obj ={
+//   id: 5,
+//   token: 12343423,
+// };
+// let keys = object.keys(obj);
```

Comment:
```suggestion
// let keys = Object.keys(obj);
```

оно не должно работать

Reply:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/43656351/164258934-fe89b6aa-ab92-46de-92e8-64d52b72a5d7.png">

Reply:
потому , что Object это глобальный конструктор в js